### DISCUSSION TITLE
Quantity type names

### DISCUSSION THREAD

**Discussion by seanpearsonuk** [ansys/pydpf-post#539] [Q&A] [answered]
[This fluids example](https://post.docs.pyansys.com/version/stable/examples/04-Fluid-Examples/00-explore-fluid-simulation.html#sphx-glr-examples-04-fluid-examples-00-explore-fluid-simulation-py) includes temperature exraction:
```
temperature = simulation.temperature()
```
The naming of such fields is constrained by DPF and I understand that providers such as the CFF (in this case) are adhering to a contract that makes this seamless. This contract comprises a set of primary field quantity types and their names. The provider can also provide arbitrarily named fields that automatically extend the `simulation` interface.

A couple of questions:
1. Is the core set of names documented?
2. Is there a naming convention for other names so that the interface looks consistent? If so, is such a convention applied at the `simulation` interface by transforming the provider's names or is that directly imposed as part of the contract with providers?

Thank you!  @cbellot000 @rlagha

**Comment by cbellot000** [comment, marked as answer]
Hi @seanpearsonuk, yes,
you can find the list of results available and wrapped in pydpf-post in the [API doc](https://post.docs.pyansys.com/version/stable/api/ansys.dpf.post.fluid_simulation.html). And in a more reusable way, taking a look at the "result"category of our [Operators doc](https://dpf.docs.pyansys.com/version/stable/operator_reference_load.html) can help, you will have the extensive list of all result available in the DPF server an by clicking on them the description of the result type

**Comment by seanpearsonuk** [comment]
@cbellot000 this is great, thanks very much!
